src/apps/components/run.js

```diff
@@ -83,21 +83,12 @@ class RunApp extends React.Component {
     render() {
         return (
             <div className="app-main">
-                <div className="main-view">
-                    <JuttleViewer bundle={this.props.bundle} />
+                <RunHeader {...this.props} onRunClick={this.handleRunClick} ref="runHeader" />
```

Comment:
I think it would be be good to explicitly pass the the props that are necessary (so its clearer what the dependencies are), or use redux's `connect` within RunHeader.